homeassistant/components/awattar/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""The aWATTar integration."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import AwattarDataUpdateCoordinator
+
+PLATFORMS = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up aWATTar from a config entry."""
+
+    coordinator = AwattarDataUpdateCoordinator(hass)
+    await coordinator.async_config_entry_first_refresh()
+
+    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
```

Comment:
Please use this pattern instead https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry